util-taglib/src/com/liferay/taglib/ui/ContentSecurityPolicyTag.java

```diff
@@ -53,6 +50,13 @@ public int doEndTag() throws JspException {
 
 	@Override
 	public int doStartTag() throws JspException {
+		if (Validator.isBlank(
+				ContentSecurityPolicyNonceProviderUtil.getNonce(
+					getRequest()))) {
```

Comment:
Not really. I do need the nonce to be present to emit the <script> node so I'm not doing this to test if CSP is enabled but to make sure I have a proper nonce available.

Additionally, we cannot check if CSP is enabled from outside of OSGi (note that this code is from `util-taglib`) because we don't have access to the needed classes.